plugins/inputs/prometheus/README.md

```diff
@@ -67,7 +67,7 @@ Measurement names are based on the Metric Family and tags are created for each
 label.  The value is added to a field named based on the metric type.
 
 All metrics receive the `url` tag indicating the related URL specified in the
-Telegraf configuration.  If using Kubernetes service discovery the `address`
+Telegraf configuration. Existing username and password for basic authentication will be removed before. If using Kubernetes service discovery the `address`
```

Comment:
Let's remove this, we shouldn't have to say this because doing it the other way would be bad.  BTW, using the userinfo section of the url is not [basic auth](https://tools.ietf.org/html/rfc7617).

Reply:
Oh sorry, you are right, this is not basic auth! I red it in the referenced issue and wasn't think about it when writing the commit message and Readme.

I'll remove the sentence in Readme.md with next commit.